src/components/views/dialogs/keybackup/RestoreKeyBackupDialog.js

```diff
@@ -184,6 +184,15 @@ export default React.createClass({
         } else if (this.state.backupInfo === null) {
             title = _t("Error");
             content = _t("No backup found!");
+        } else if (this.state.recoverInfo && this.state.recoverInfo.imported === 0) {
+            title = _t("Error Restoring Backup");
+            let failedToDecrypt;
+            content = <div>
+                <p>{_t(
+                    "Failed to decrypt any sessions: check that the recovery key " +
```

Comment:
Hmm, this string feels a bit too technical too me... Perhaps something like: "Riot wasn't able to decrypt anything from your key backup. Please verify that you entered the correct recovery key."

Reply:
yeah, good point - have simplified the first bit further to, "failed to restore backup".